.all-contributorsrc

```diff
@@ -827,6 +827,13 @@
       ]
     },
     {
+      "login": "ooHAoo",
+      "name": "Huda Al Dallal",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/26858034?s=400&u=fd8893b7f81dd9c9baeaf2807a1f7f98db0b2fe2&v=4",
+      "profile": "https://github.com/ooHAoo",
+      "contributions": [
+        "translation",
+        "tool"
```

Comment:
I saw `tool` rarely. Can you explain a little more about it?

Reply:
This was added accidentally and I thought I did fix it.
Sorry about that, will remove that once again.